src/components/Filters/StatefulFilters.tsx

```diff
@@ -129,21 +140,26 @@ export class StatefulFilters extends React.Component<StatefulFiltersProps, State
       value: value
     };
 
-    const typeFilterPresent = activeFilters.filter(filter => filter.category === field.title).length > 0;
+    const typeFilterPresent = activeFilters.filters.filter(filter => filter.category === field.title).length > 0;
 
     if (field.action === FILTER_ACTION_UPDATE && typeFilterPresent) {
-      activeFilters.forEach(filter => {
+      activeFilters.filters.forEach(filter => {
         if (filter.category === field.title) {
           filter.value = value;
         }
       });
     } else {
-      activeFilters.push(activeFilter);
+      activeFilters.filters.push(activeFilter);
     }
 
     this.updateActiveFilters(activeFilters);
   };
 
+  updateOpSortLabel = (value: LabelOperation) => {
+    this.setState({ activeFilters: { filters: this.state.activeFilters.filters, op: value } });
```

Comment:
this `setState` update is performed twice: first here, then within the `updateActiveFilters` call below. So I guess the first can be removed?

Reply:
mmm you're right  I'll remove it